app/scheduler/modules/pairing.py

```diff
@@ -0,0 +1,111 @@
+"""Match two Launch Pad member for a private conversation"""
+from slack import WebClient
+from interface.slack import Bot
+from random import shuffle
+from .base import ModuleBase
+from typing import Dict, List, Any, Set
+from flask import Flask
+from config import Config
+from db.facade import DBFacade
+from app.model import Pairing
+import logging
+
+
+class PairingSchedule(ModuleBase):
+    """Module that matches 2 Launch Pad members each week"""
+
+    NAME = 'Match launch pad members randomly'
+
+    def __init__(self,
+                 flask_app: Flask,
+                 config: Config,
+                 facade: DBFacade):
+        """Initialize the object."""
+        self.bot = Bot(WebClient(config.slack_api_token),
+                       config.slack_notification_channel)
+        self.channel_id = self.bot.get_channel_id(config.slack_pairing_channel)
+        self.facade = facade
+
+    def get_job_args(self) -> Dict[str, Any]:
+        """Get job configuration arguments for apscheduler."""
+        return {'trigger':      'cron',
+                'minute': '*',
+                'name':         self.NAME}
+
+    def do_it(self):
+        """Pair users together, and create a private chat for them"""
+        users = self.bot.get_channel_users(self.channel_id)
+        logging.debug(f"users of the pairing channel are {users}")
+        matched_user_pairs = self.__pair_users(users)
+        for pair in matched_user_pairs:
+            group_name = self.bot.create_private_chat(pair)
+            logging.info(f"The name of the created group name is {group_name}")
+            self.bot.send_to_channel("Hello! \
+                You have been matched by Rocket \
+                please use this channel to get to know each other!",
+                                     group_name)
```

Comment:
I wonder if we should have a configuration option `SLACK_PAIRING_TIPS_LINK` that can generate a button that links to tips for what to do in a pairing (i.e. "set up video call, talk about x and y") - for Launch Pad this could be a handbook page, for example

Reply:
Hmmm Can we do this on a follow up or do ya think it's important? Could be a really good `first issue` for someone!

Reply:
Yep! Can you add all the TODO items for follow-up issues in the PR description as a checklist, to make sure we don't forget?